### PR TITLE
Annie/doc guidelines

### PR DESCRIPTION
**Pull request by AnnieLR**
First draft. Added directly to docs folder. 
Realized I branched this off the camelCase branch. Don't think it's a problem cuz it's its own file?

### CODE REVIEW

File: site/docs/guides/component-variants.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Component Variants
 layout: docs
 ---
 
-When building a [Stylable component library](./stylable-component-library.md) or a [Stylable application](./stylable-application.md), it is useful to define several semantic "flavors" of some components, for example a `Button` component that represents `cancel` or a `Gallery` component for the `main` gallery instance. 
+When building a [Stylable component library](./stylable-component-library.md) or a [Stylable application](./stylable-application.md), it is useful to define several semantic "flavors" of some components, for example a `button` component that represents `cancel` or a `gallery` component for the `main` gallery instance.
```
Comment (arnonkehat) [inline site/docs/guides/component-variants.md]: this change is in 2 different pull requests, and wrong in both. why?

File: site/docs/guides/components-basics.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ This guide walks you through the basics of how to style and work with components
 
 You use **Stylable** with a component file (for example using React), along with a **Stylable** CSS file that has the extention `.st.css`.
 
-> **Note**:
+> **Note**:  
 > This guide shows the JSX side of our [stylable-integration](https://github.com/wixplosives/stylable-integration) with React. 
 
-**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `Button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
+**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
```
Comment (arnonkehat) [inline site/docs/guides/components-basics.md]: same here

File: site/docs/guides/components-basics.md [hunk 3]
```diff
@@ -8,17 +8,17 @@ This guide walks you through the basics of how to style and work with components
...
 
 Whether creating your own components or using components you imported from a 3rd party, you want to be able to access and style the internal parts of every component in the scope of your page or application. 
 
 
-## 1 Style a Component 
+## 1 Style a component 
 
-Let's say you have a `Button` component with a render function per this example. You can style its JSX using the `className` property.
+Let's say you have a `button` component with a render function per this example. You can style its JSX using the `className` property.
```
Comment (arnonkehat) [inline site/docs/guides/components-basics.md]: capitalize.

**Comment by AnnieLR** [comment]
Deleting this branch. As I wrote, based this on another branch so deleting PR and branch.

**Pull request closed without merging**